### PR TITLE
add DeFiner fixed price oracle for token TPT

### PR DESCRIPTION
**Pull request by jchittoda**

### CODE REVIEW

File: contracts/oracle/definer-oracle/ExOracleTPT.sol [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+    // Price Type required by ExOracle. Example: "DAI"
+    string public constant priceType = "TPT";
+
+    uint256 public constant ONE_OKT = 10 ** 18;
+    // 6 decimals USD rate returned by ExOracle, multiplier, divisor
+    uint256 public constant USD_DECIMALS_MUL_DIV = 10 ** 6;
+    uint256 public constant OKT_NUMERATOR = ONE_OKT * USD_DECIMALS_MUL_DIV;
+
+    event PriceUpdated(uint256 priceInUSD, uint256 timestamp);
```
Comment (Rishabh42) [inline contracts/oracle/definer-oracle/ExOracleTPT.sol:37]: I think this event should be called `TPTPriceUpdated`, so we don't confuse while adding more tokens in the future

File: contracts/oracle/definer-oracle/ExOracleTPT.sol [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+    /**
+     * @dev Only owner can set the TPT price in USD
+     * @notice The price is in USD, upto 6 decimal places.
+     * for example to set $1.1, set `_price = 1100000`
+     */
+    function setTPTPriceInUSD(uint256 _price) public onlyOwner {
+        tptPriceInUSD = _price;
+        tptLastUpdateTimestamp = now;
+        emit PriceUpdated(tptPriceInUSD, tptLastUpdateTimestamp);
```
Comment (Rishabh42) [inline contracts/oracle/definer-oracle/ExOracleTPT.sol:60]: same change `TPTPriceUpdated`

**Review by Rishabh42 — COMMENTED** [review]
Just a minor change, rest looks okay

File: contracts/oracle/definer-oracle/ExOracleTPT.sol (hunk 1 above)
Comment (jchittoda) [reply contracts/oracle/definer-oracle/ExOracleTPT.sol]: fixed

File: contracts/oracle/definer-oracle/ExOracleTPT.sol (hunk 2 above)
Comment (jchittoda) [reply contracts/oracle/definer-oracle/ExOracleTPT.sol]: fixed

**Review by Rishabh42 — APPROVED** [review]

**Pull request merged**